extensions/hibernate-reactive/deployment/src/main/java/io/quarkus/hibernate/reactive/deployment/HibernateReactiveProcessor.java

```diff
@@ -83,6 +85,8 @@ void registerBeans(BuildProducer<AdditionalBeanBuildItem> additionalBeans, Combi
             List<PersistenceUnitDescriptorBuildItem> descriptors,
             JpaModelBuildItem jpaModel) {
         if (!hasEntities(jpaModel)) {
```

Comment:
There's something wrong in the code **before this PR**: this condition seems pointless, because if the JPA model doesn't contain any entity, we don't create any `PersistenceUnitDescriptorBuildItem`, and the `if` block below won't get executed anyway. See the guard (`if(!hasEntities(jpaModel)) { return; }`) in `io.quarkus.hibernate.reactive.deployment.HibernateReactiveProcessor#buildReactivePersistenceUnit` and `io.quarkus.hibernate.orm.deployment.HibernateOrmProcessor#configurationDescriptorBuilding`.

So (before your changes), we could just remove this `if`, and it wouldn't change anything to this code.

So I think this log should rather be in an `else` condition to the `if` below? That would also have the advantage of logging something if we're skipping bean registration because there are more than one persistence unit.

(Why the hell do we do that, by the way? We don't do that for Hibernate ORM).